docs/manual/advanced-features.tex

```diff
@@ -1006,6 +1006,17 @@
 the method has no side effects, so calling it does not invalidate any
 dataflow facts.
 
+The \refqualclass{dataflow/qual}{SideEffectsOnly} annotation
+specifies the superset of expressions that the method might side effect.
+So expressions that are not specified as annotation values of
```

Comment:
Don't start a sentence with a joining word like "So".  It is informal and it's poor grammar.  (It only makes the documentation a little bit harder to read, but you should excise anything that makes the documentation even a little bit harder to read.)